integration-tests/features/dependency_editor_categories.feature

```diff
@@ -0,0 +1,50 @@
+Feature: Dependency Editor Categories Search API behaviour
+
+  Scenario: Check that the API entry point
+    Given System is running
+    When I access /api/v1/categories/springboot
+    Then I should get 401 status code
+
+
+  @requires_authorization_token
+  Scenario: Check the categories search for existing runtime springboot from the maven ecosystem
+    Given System is running
+    When I acquire the authorization token
+    Then I should get the proper authorization token
+    When I search for the runtime springboot
+    Then I should get 200 status code
+     And I should receive JSON response with the request_id
+     And I should find categories
+     And I should find categories with pkg_count and list of packages 
```

Comment:
How difficult would it be to extract the set of packages returned - and compare it with a list of expected packages? Do you think that this will be overkill for a smoke test?

Reply:
No, It won't
```python
categories = data.get('categories', {})
assert categories
category = list(categories)[0]
pkg_count = categories[category].get('pkg_count')
assert pkg_count
packages = categories[category].get('packages')
assert packages
```

Reply:
IMHO we need to test specific packages, it won't be too difficult to update tests later